src/wwwroot/app/src/scripts/__ClassroomJoin.js

```diff
@@ -0,0 +1,87 @@
+var tempsid = $('#tempsid').val();
+
+var colorArray = [
+    "red darken-1", "teal darken-1", "indigo darken-1", "blue-grey darken-1",
+    "purple darken-3", "blue lighten-1", "blue accent-2", "light-blue"
+];
+
+$(document).ready(function () {
+    getClassRoom();
+$("#classroomForm").submit(function (e) {
+    e.preventDefault();
+    var AccessCode = $('#AccessCode').val();
+    var studentId = $("#student_id").val();
+    console.log("access code: "+AccessCode);
+
+    $.post('', {accessCode : AccessCode, sid : studentId}, function (responseData) {
+        if(responseData === "failed"){
+            showMaterialToast("Invalid Request", "red darken-1");
+            $('#AccessCode').val("");
+        }else if(responseData === "codeinvalid"){
+            showMaterialToast("Enter a valid classroom code", "amber darken-3");
+            $('#AccessCode').val("");
+        }
+        else if(responseData === "success"){
+            showMaterialToast("Classroom join Successfully", "teal darken-1");
+
+            $('#AccessCode').val("");
+            getClassRoom();
+        }else{
+
+        }
+
+        console.log(responseData);
+    });
+})
+});
+
+function showMaterialToast(data, style) {
+    M.toast({
+        html : data,
+        classes : style
+    });
+}
+
+function getClassRoom() {
+
+    var classrooms = $('#classRooms');
+    console.log("sid is : " + tempsid);
+    console.log("classroom is : " + classrooms);
+
+    $.get('/Students/__getClassRoom___', {sid : tempsid}, function (res) {
+        if(res.length == 0){
+            var modalForCreateNewClassroom = "<h5 class=\"center-align\">You have no classroom yet</h5>\n" +
```

Comment:
Change the variable name. It should be modalToJoinNewClassroom. Carefully change this variabe in other places where used. Try finding the word with Ctrl + F